scipy/optimize/_hessian_update_strategy.py

```diff
@@ -0,0 +1,307 @@
+"""Implementation of Quasi-Newton update strategies."""
```

Comment:
I would simplify to `Quasi-Newton Hessian update strategies.`

Reply:
Ok!